Dockerfile

```diff
@@ -21,15 +21,31 @@ WORKDIR /usr/src/minetest
 RUN apk add --no-cache git build-base irrlicht-dev cmake bzip2-dev libpng-dev \
 		jpeg-dev libxxf86vm-dev mesa-dev sqlite-dev libogg-dev \
 		libvorbis-dev openal-soft-dev curl-dev freetype-dev zlib-dev \
-		gmp-dev jsoncpp-dev postgresql-dev && \
+		gmp-dev jsoncpp-dev postgresql-dev ca-certificates && \
 	git clone --depth=1 -b ${MINETEST_GAME_VERSION} https://github.com/minetest/minetest_game.git ./games/minetest_game && \
-	rm -fr ./games/minetest_game/.git && \
+	rm -fr ./games/minetest_game/.git
+
+WORKDIR /usr/src/
+RUN git clone --recursive https://github.com/jupp0r/prometheus-cpp/ && \
+	mkdir prometheus-cpp/build && \
+	cd prometheus-cpp/build && \
+	cmake .. \
+		-DCMAKE_INSTALL_PREFIX=/usr/local \
+		-DCMAKE_BUILD_TYPE=Release \
+		-DENABLE_TESTING=0 && \
+	make -j2 && \
+	make install
+
+WORKDIR /usr/src/minetest
+# Enable metrics listener on a nice address for Docker environments
+RUN sed -i 's/127.0.0.1:8080/0.0.0.0:30000/' src/util/metricsbackend.cpp src/defaultsettings.cpp && \
```

Comment:
this isn't very portable, but no idea if there's a better solution

Reply:
as it's properly exposed in the dockerfile that should not be a problem. it ensure that you can setup a docker volume with a minetest.conf without disabling metrics by error